tools/release-notes/github.go

```diff
@@ -42,7 +42,12 @@ func (p *project) gitHubGetReleaseNote(number int) (string, error) {
 	for i, line := range body {
 		note, err := gitHubParseReleaseNote(i, line, body)
 		if err == nil {
-			note = fmt.Sprintf("[PR #%d][%s] %s", number, *pr.User.Login, note)
+			if len(p.repoPrettyUrl) > 0 {
```

Comment:
Always true? in tools/release-notes/release-notes.go:240, the `repoPrettyUrl` field is always set.

Reply:
it's set for all but the HCO repo

Reply:
done